dashboard/src/components/api/devfile-registry.factory.ts

```diff
@@ -58,25 +58,42 @@ export class DevfileRegistry {
     this.jwtproxyMemoryLimitNumber = this.getMemoryLimit(DEFAULT_JWTPROXY_MEMORY_LIMIT);
   }
 
+  private getRegistryAuthorization(): string {
+    // TODO add an ability to configure registry as secure
+    return `Basic ${btoa('Che devfile')}`;
```

Comment:
hello, why do we send a dummy token instead of no auth headers for now ?

Reply:
In the issue https://github.com/eclipse/che/issues/14627 we decide to provide an ability to configure plugin/devfile registries as secure. That is why I added dummy implementation as a first step.

Reply:
fixed